source/global.cc

```diff
@@ -100,6 +100,4 @@ namespace aspect
     }
   }
 
-  // Output parallel statistics [bool]
-  const bool output_parallel_statistics = false;
```

Comment:
output_parallel_statistics seems to have just been set by hand in the code, no associated parameter.